### DISCUSSION TITLE
Happy Birthday Mischa

### DISCUSSION THREAD

**Discussion by daltonfury42** [SimplQ/simplQ-frontend#568] [General]
Hey..

LinkedIn tells me today is your birthday. Happy Birthday 🎊 

![tenor](https://user-images.githubusercontent.com/11256376/110092966-34ebe600-7dc0-11eb-812f-52b9212d9229.gif)

**Comment by maaverik** [comment]
Fun fact: It's @daltonfury42's birthday today too :). Happy birthday to the both of you!

**Reply by mradenovic** [reply]
Happy birthday to us! 🤣

**Reply by daltonfury42** [reply]
😝
